libs/utils/utils.js

```diff
@@ -455,9 +455,13 @@ export function decorateLinks(el) {
       a.setAttribute('target', '_blank');
       a.href = a.href.replace('#_blank', '');
     }
-    const autoBLock = decorateAutoBlock(a);
-    if (autoBLock) {
-      rdx.push(a);
+    if (a.href.includes('#_dnb')) {
+      a.href = a.href.split('#')[0];
+    } else {
+      const autoBLock = decorateAutoBlock(a);
```

Comment:
Nit: I noticed the repeated uppercase in `autoBLock` when I reviewed this code in a different context. Since you're already touching this code, would you mind correcting this to `autoBlock`? Thx :)

Reply:
I have fixed this.